realwork/WatchApp Extension/NotificationController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NotificationController.swift
+//  WatchApp Extension
+//
+//  Created by Nathan Ngobale on 2020/04/29.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import WatchKit
+import Foundation
+import UserNotifications
+
+
+class NotificationController: WKUserNotificationInterfaceController {
+
+    override init() {
+        // Initialize variables here.
+        super.init()
+        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5444460)